packages/crypto/test/pbkdf2.test.ts

```diff
@@ -0,0 +1,58 @@
+import { bufferFromString, pbkdf2 } from '..';
+
+/**
+ * @group node
+ * @group browser
+ */
+describe('pbkdf2 node & browser', () => {
+  it('should compute the PBKDF2 hash correctly', () => {
+    const passwordBuffer = bufferFromString(String('password123').normalize('NFKC'), 'utf-8');
+    const saltBuffer = bufferFromString(String('salt456').normalize('NFKC'), 'utf-8');
+    const iterations = 1000;
+    const keylen = 32;
+    const algo = 'sha256';
+    const expectedResult = '0x90eceedd899d5cdcdfd9b315ad6e2c3391bf95cc131b6f0f016339db5ee60494';
+
+    const result = pbkdf2(passwordBuffer, saltBuffer, iterations, keylen, algo);
+
+    expect(result).toBe(expectedResult);
+  });
+});
+
+// #TODO: Even when I run the filter:browser, these tests still run
```

Comment:
I'm skipping these are they are not critical tests but locally even when I run the browser filter these tests still run, could it be because they are in the same file?

Reply:
That's odd 🤔 . 

But I reckon you are right, maybe this unintended behavior is happening because they are within the same file

Reply:
I decided to move them to seperate files for now and then we can re-adjust the script to be more flexible in a follow up PR if needs be.